.github/workflows/inductor_xpu_ut_nightly.yml

```diff
@@ -0,0 +1,203 @@
+name: inductor-xpu-ut-test
+
+on:
+  workflow_call:
+    inputs:
+      torch_xpu_ops_update:
+        required: false
+        type: string
+        default: 'true'
+        description: True means update xpu_ops when building pytorch, otherwise means not
+      expected_fail_num:
+        required: false
+        type: number
+        description: for UT result check
+
+jobs:
+  Inductor-XPU-UT-Tests:
+    runs-on: linux.idc.xpu
```

Comment:
please also add this runner label as this callable workflow inputs, default `linux.idc.xpu`

Reply:
Done